jep/200/README.adoc

```diff
@@ -384,11 +384,18 @@ N/A
 
 Current state:
 
-* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
-* link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)] has been executed for a limited plugin set
+* link:https://github.com/jenkinsci/acceptance-test-harness[Jenkins Acceptance Test Harness (ATH)] has been executed with the patched components, several plugins were fixed (see the _Testing_ section)
+** After the discoveries in PCT (see below), the ATH results need to be reviewed carefully
+** BDFL Delegate is waiting for info about what has been tested
+(Jenkins WAR from link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] or a custom build with custom plugin versions)
```

Comment:
Custom build using the source branches from that PR. Not sure what difference it makes.

Reply:
Discussion summary:

* Some tested plugins were not at the latest version. In particular, Maven plugin is not 3.0
* Custom build alters Jenkins behavior a bit, but these changes should not impact XStream/Remoting serialization logic in tests

We will double-check which plugins need to be retested